src/main/convert.js

```diff
@@ -98,10 +100,19 @@ export class ThriftFileConverter {
   generateTypedef = (def: Typedef) =>
     `export type ${this.transformName(def.id.name)} = ${this.types.convert(def.valueType)};`;
 
-  generateEnum = (def: Enum) =>
-    `export type ${this.transformName(def.id.name)} = ${def.definitions
-      .map(d => `"${d.id.name}"`)
-      .join(' | ')};`;
+  generateEnum = (def: Enum) => {
+    if (this.enumvalues) {
+      return `export type ${this.transformName(def.id.name)} = number;
+       export type ${this.transformName(def.id.name)}Keys = ${def.definitions
+        .map(d => `"${d.id.name}"`)
+        .join(' | ')};`;
+    } else {
+      return `export type ${this.transformName(def.id.name)}Values = number;
+       export type ${this.transformName(def.id.name)} = ${def.definitions
```

Comment:
Is it possible to do a '|' of the numeric values? Like `type MyEnum = 1 | 3 | 4 | 5`? Is that silly?